tests/xgov_registry/test_request_unsubscribe_xgov.py

```diff
@@ -0,0 +1,233 @@
+    app_xgov_managed_subscription: XGovSubscriberAppMockClient,
+) -> None:
+    xgov_fee = get_xgov_fee(xgov_registry_client)
+    xgov_registry_client.send.pause_registry()
+    with pytest.raises(LogicError, match=err.PAUSED_REGISTRY):
+        xgov_registry_client.send.request_unsubscribe_xgov(
+            args=RequestUnsubscribeXgovArgs(
+                xgov_address=app_xgov_managed_subscription.app_address,
+                owner_address=deployer.address,
+                relation_type=0,
+                payment=algorand_client.create_transaction.payment(
+                    PaymentParams(
+                        sender=deployer.address,
+                        receiver=xgov_registry_client.app_address,
+                        amount=xgov_fee,
+                    )
+                ),
+            ),
+        )
+
+    xgov_registry_client.send.resume_registry()
+    xgov_registry_client.send.request_unsubscribe_xgov(
+        args=RequestUnsubscribeXgovArgs(
+            xgov_address=app_xgov_managed_subscription.app_address,
+            owner_address=deployer.address,
+            relation_type=0,
+            payment=algorand_client.create_transaction.payment(
+                PaymentParams(
+                    sender=deployer.address,
+                    receiver=xgov_registry_client.app_address,
+                    amount=xgov_fee,
+                )
+            ),
+        )
+    )
+
+
+def test_request_unsubscribe_xgov_locked(
+    algorand_client: AlgorandClient,
+    deployer: SigningAccount,
+    xgov_registry_client: XGovRegistryClient,
+    app_xgov_managed_subscription: XGovSubscriberAppMockClient,
+) -> None:
+
+    # Lock voting address
+    xgov_box = xgov_registry_client.send.get_xgov_box(
+        args=GetXgovBoxArgs(xgov_address=app_xgov_managed_subscription.app_address)
+    ).abi_return
+    xgov_registry_client.send.set_voting_account(
+        args=SetVotingAccountArgs(
+            xgov_address=app_xgov_managed_subscription.app_address,
+            voting_address=app_xgov_managed_subscription.app_address,
+        ),
+        params=CommonAppCallParams(sender=xgov_box.voting_address),
+    )
+
+    # Request managed unsubscription via owner
+    initial_request_id = xgov_registry_client.state.global_state.request_id
+    xgov_registry_client.send.request_unsubscribe_xgov(
+        args=RequestUnsubscribeXgovArgs(
+            xgov_address=app_xgov_managed_subscription.app_address,
+            owner_address=deployer.address,
+            relation_type=0,
+            payment=algorand_client.create_transaction.payment(
+                PaymentParams(
+                    sender=deployer.address,
+                    receiver=xgov_registry_client.app_address,
+                    amount=get_xgov_fee(xgov_registry_client),
+                )
+            ),
+        )
+    )
+    final_request_id = xgov_registry_client.state.global_state.request_id
+    assert final_request_id == initial_request_id + 1
```

Comment:
@uhudo can you explain this test condition?

Reply:
This test is a bit redundant. It was to show the issue with the initial xGov version where an account of a managed subscription can be locked by setting the `voting_address` to e.g. `xgov_address`. This case just explicitly shows that the `owner_address` can still request a managed unsubscription even in such a situation.

Reply:
Got it! Perhaps a brief comment in the test could be useful for future readers, as the test's scope is not immediately evident.

Reply:
Added a brief comment in [cc9ed37](https://github.com/algorandfoundation/xgov-beta-sc/pull/314/commits/cc9ed378a314779acb8350f287f7b470b9b48441).